Cargo.toml

```diff
@@ -382,6 +382,15 @@ members = [
     # http://rosettacode.org/wiki/Range_extraction
     "tasks/range-extraction",
 
+    # http://rosettacode.org/wiki/Anonymous_recursion
+    "tasks/recursion/anonymous-recursion",
```

Comment:
The task name is "Anonymous recursion", so it shouldn't be grouped under a "recursion" folder. Subfolders are meant for tasks with names like "MD5/Implementation". Ditto for the other tasks.